modules/cli-installing-cli.adoc

```diff
@@ -38,25 +38,93 @@ of the commands in {product-title} {product-version}. Download and
 install the new version of `oc`.
 ====
 
+[id="cli-installing-cli-on-linux_{context}"]
+== Installing the CLI on Linux
+
+You can install the OpenShift CLI (`oc`) binary on Linux using the following procedure.
+
 .Procedure
+
 ifdef::openshift-origin[]
-. Navigate to https://mirror.openshift.com/pub/openshift-v4/clients/oc/latest/ and choose the folder for your operating system
-. Download `oc.tar.gz`
+. Navigate to https://mirror.openshift.com/pub/openshift-v4/clients/oc/latest/ and choose the folder for your operating system and architecture.
```

Comment:
s/Navigate to https://mirror.openshift.com/pub/openshift-v4/clients/oc/latest/ and choose the/Navigate to the https://mirror.openshift.com/pub/openshift-v4/clients/oc/latest/ page, and then choose the/